### DISCUSSION TITLE
blobstore 纠删码系统是否支持原始数据，校验数据独立存储？

### DISCUSSION THREAD

**Discussion by fengjiannan2010** [cubefs/cubefs#3391] [Q&A] [answered]
blobstore 纠删码系统是否支持原始数据，校验数据独立存储？
比如数据a,b,c,d
校验数据e

a,b,c,d存储在一个磁盘HDD1
e单独存储在另一个磁盘HDD2

基于以上我想在蓝光存储中实现数据盘，校验盘分区存储。

**Comment by JasonHu520** [comment, marked as answer]
当前不会区分存储介质，都是随机打散
